controllers/trustyaiservice_controller.go

```diff
@@ -44,7 +44,7 @@ const (
 	defaultTag           = string("latest")
 	containerName        = "trustyai-service"
 	serviceMonitorName   = "trustyai-metrics"
-	finalizerName        = "trustyai.opendatahub.io/finalizer"
+	finalizerName        = "trustyai.opendatahub.io.trustyai.opendatahub.io/finalizer"
```

Comment:
Is it possible to use a shared constant to avoid/prevent similar issues?

Reply:
- https://github.com/trustyai-explainability/trustyai-service-operator/issues/73